src/scikit_package/app.py

```diff
@@ -14,20 +16,22 @@
 exist_config = config_file.exists()
 
 
-def create(package_type):
-    if package_type == "workspace":
+def create(entry_type):
```

Comment:
Rename from `package_type` to `entry_type` since we are also introducing "paper"